projects/storefrontapp/src/environments/environment.ts

```diff
@@ -23,4 +23,5 @@ export const environment: Environment = {
   cpq: buildProcess.env.CX_CPQ ?? false,
   digitalPayments: buildProcess.env.CX_DIGITAL_PAYMENTS ?? false,
   epdVisualization: buildProcess.env.CX_EPD_VISUALIZATION ?? false,
+  pickupInStore: buildProcess.env.CX_PICKUP_IN_STORE ?? false,
```

Comment:
Default to true but leave a todo to change to false.

Reply:
Done